src/app/contexts/RequestContext/isNotEEA/index.js

```diff
@@ -0,0 +1,52 @@
+const eea = [
```

Comment:
Is eea an acronym for something?

Reply:
Yes, it's https://en.wikipedia.org/wiki/European_Economic_Area

Reply:
I think adding a  comment would be helpful to anyone looking at this.

Reply:
Done! Thanks @sadickisaac